server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java

```diff
@@ -108,15 +116,18 @@ default void getUsersQuotaShouldReturnAllUsersWhenNoParameters(WebAdminQuotaSear
 
             testSystem.getQuotaSearchTestSystem().await();
 
-            given()
-                .get("/quota/users")
-            .then()
-                .statusCode(HttpStatus.OK_200)
-                .contentType(ContentType.JSON)
-                .body("username", containsInAnyOrder(
-                    BOB.asString(),
-                    JACK.asString(),
-                    GUY_WITH_STRANGE_DOMAIN.asString()));
+            CALMLY_AWAIT.atMost(Duration.ofSeconds(10))
+                .untilAsserted(() -> {
+                    given()
+                        .get("/quota/users")
+                        .then()
```

Comment:
Indent

Reply:
I will give a few more details of Lan comment that lacks precisions (habits)

When testing REST calls we align given/when/then/with at the same indent level to improve test readaiblity (like on the old diff).

You likely should preserve this indentation property, here and in other places.

Reply:
fixed